src/release.js

```diff
@@ -86,6 +93,8 @@ module.exports = async function ({ github, context, inputs }) {
       await tagVersionInGit(`v${major}`)
       await tagVersionInGit(`v${major}.${minor}`)
       await tagVersionInGit(`v${major}.${minor}.${patch}`)
+    } else {
+      logInfo('missing sync-semver')
```

Comment:
I think we should use `logError` here if something as crucial as semver is missing.